spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -83,4 +83,58 @@
       end
     end
   end
+
+  describe 'POST #create' do
+    context 'given an invalid file' do
+      it 'does not save any new keyword' do
+        create_token_header
```

Comment:
Should we do something similar to `authenticated_user: true`, something like

```
   it `does not save any new keyword`, authenticated_api_user: true do
  end
```

Reply:
It won't cover all tests (as some needs to assign keywords to the user). 
But it's indeed a good option.
Note that I had to use `:each` instead of `:all` in here: `config.before(:each, ...`. 
The reason is that this helper edits a context-specific object (`request`) to insert the authorization header. 

Working well and overall cleaner than before :) 

**Additional insight:** While it's possible to go further with a helper like `authenticated_api_user: :quantity_keywords` and generate keywords too, I think it brings too much complexity for the context of a test (considering the benefits). 
So I would not go that far at the moment. 
Let me know if you think differently @ankitkalia1195  :) 

Thanks!